hoodie-utilities/src/main/java/com/uber/hoodie/utilities/config/AbstractCommandConfig.java

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
Also, the license is now  changed to Apache with no Uber copyright. Can you copy from a diff file.